pages/common/pv.md

```diff
@@ -0,0 +1,27 @@
+# pv
+
+> Monitor the progress of data through a pipe.
+
+- Print the contents of the file and display a progress bar:
+
+`pv {{file}}`
+
+- Measure the speed and amount of data flow between pipes (`-s` is optional):
+
+`command1 | pv -s {{expected_amount_of_data_for_eta}} | command2`
+
+- Recompress a file, also see the uncompressed and recompressed file size:
+
+`pv -cN gz file.gz | gzcat | pv -cN uncompressed | xz | pv -cN xz > out.xz`
```

Comment:
What does `-cN` do? Does it make sense to have 2 examples instead for `-c` and `-N`?